packages/eslint-plugin-react-hooks/__tests__/ESLintRuleExhaustiveDeps-test.js

```diff
@@ -1264,17 +1302,16 @@ const tests = {
       errors: [
         {
           message:
-            "React Hook useCallback has a missing dependency: 'props.foo?.toString'. " +
+            "React Hook useCallback has a missing dependency: 'props.foo.toString'. " +
```

Comment:
This is the regression. But it's acceptable IMO because it only affects the message and the autofix. All other regression tests that have been added are passing.

Reply:
In fact this test is also kind of wrong because it suggests `props.foo?.toString` should be added — whereas in practice it should have suggested `props.foo`. So this is something we need to fix too.

Reply:
Pushed a fix to this test.